src/FuncSharp/Strings/StringExtensions.cs

```diff
@@ -5,13 +5,19 @@ namespace FuncSharp
 {
     public static class StringExtensions
     {
-        public static IOption<string> ToNonEmptyOption(this string s)
+        public static bool IsEmptyOrWhiteSpace(this string s)
```

Comment:
dont know what this method is good for, honestly :-)

Reply:
Well it's just sugar syntax. You're right, no need to have it in funcsharp

Reply:
I moved it cause it was used by the NonEmptyString.Create